tests/test_sd_whonix.py

```diff
@@ -5,6 +5,18 @@
 from base import SD_VM_Local_Test
 
 
+def skip_if_v3_is_missing():
+    """
+    Returns True if v3 address is not setup
+    """
+    with open("config.json") as c:
+        config = json.load(c)
+        if config["hidservv3"]["hostname"] == "ADD_YOUR_VALUE":
```

Comment:
If I am not mistaken, a user copy pasting or using their existing `config.json` (without `hidservv3`) on this branch will cause the tests to fail due to keyerror on hidservv3? 

I don't think we should expect end-users to reconcile their `config.json` with `config.json.example` every time we introduce new configurations generally. What do you think?